src/dmd/backend/dwarf.d

```diff
@@ -16,7 +16,7 @@ extern (C++):
 
 nothrow:
 
-enum DWARF_VERSION = 3;
+enum DWARF_VERSION = 5;
```

Comment:
You probably want to keep this to 4 for the time being, then allow emitting version 5 code as experimental.

Reply:
Should I make a compiler switch ?
Something like `--dwarf=5` ?